record.ts

```diff
@@ -22,15 +22,25 @@ const compareStrings = toCompare(ordString);
 
 const sortStrings = (keys: string[]): string[] => keys.sort(compareStrings);
 
+/**
+ * Creates a new object with the same keys of `ta`. Values are transformed
+ * using `fai`.
+ *
+ * @example
```

Comment:
nitpick (non-blocking): 
@baoshan we don't currently use the @example tag anywhere for examples, they're simply fenced in a code block with `ts` flag

@baetheus we should decide for one style, either using @example tag for all examples, or none

That is, my work on #47 currently depends on the fact that it's a code fence block and I'm not parsing tagged blocks. By no means it's fixed as I only have some POC code atm, regardless, I think we should be consistent about the way we produce examples.

Reply:
Resolved: as it's non blocking, this is more for discussion than blocking the PR.

Reply:
I don't think there is any syntax parsing in deno for the ```@example``` tag. However, I don't think it matters either way. For consistency, let's start using @example even though it isn't specified in https://deno.land/manual@v1.24.3/testing/documentation#documentation-tests 

It's not necessary to make a repo-wide change unless either of you feel so inclined. We can be inconsistent here until someone with more desire for perfect styling than I comes along with a PR.